java-manta-cli/src/main/java/com/joyent/manta/cli/MantaCLI.java

```diff
@@ -295,6 +298,11 @@ public void run() {
         @CommandLine.Parameters(index = "2", description = "path to write the key to")
         private Path path;
 
+        @SuppressWarnings("unused")
+        @CommandLine.Option(names = {"-key", "--set-env"}, description = "configure environment "
```

Comment:
This doesn't actually seem to persist the value to the environment unfortunately:
```
[kelly@mantadev java-manta]$ java -jar /home/kelly/.m2/repository/com/joyent/manta/java-manta-cli/3.4.4-SNAPSHOT/java-manta-cli-3.4.4-SNAPSHOT-jar-with-dependencies.jar generate-key AES 256 java.256 --set-env 

Generating key
Writing [AES-256] key to [java.256]
Configure java-manta property:[MANTA_ENCRYPTION_KEY_PATH] with value:[java.256]

[kelly@mantadev java-manta]$ env | grep MANTA_ENCRYPTION_KEY_PATH

```

I'd say just take this option out since it doesn't really work for the intended purpose.

Reply:
Done and deleted.